packages/drivers/routerlicious-driver/src/documentDeltaConnection.ts

```diff
@@ -107,4 +107,15 @@ export class R11sDocumentDeltaConnection extends DocumentDeltaConnection {
 			url: getUrlForTelemetry(this.url, socketIoPath),
 		};
 	}
+
+	/**
+	 * Disconnect from the websocket
+	 */
+	protected disconnectCore(err: IAnyDriverError): void {
+		if (this.hasDetails) {
+			// tell the server we are disconnecting this client from the document
+			this.socket.emit("disconnect_document", this.clientId, this.documentId, err);
+		}
```

Comment:
If we don't have details, might we still have a disconnect reason?
We cannot send the error - it is too dangerous for privacy.
We can use something different than SPE. Should confer with AFR for what is most useful.

Reply:
removed condition